src/screens/EthToSub.tsx

```diff
@@ -105,36 +106,33 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 		}
 
 		let unsubscribe: () => void;
-		if (receiver) {
-			const recv = parseReceiver(receiver);
-			api.derive.balances.account(recv,
+		if (receiverAddress) {
+			api.derive.balances.account(receiverAddress,
 				data => setReceiverBalance( data.freeBalance.toString()))
 				.then( unsub => {unsubscribe = unsub;})
 				.catch(console.error);
 		}
 
 		return () => unsubscribe && unsubscribe();
-	}, [api, isApiReady, receiver]);
-
-	const changeReceiver = (_: any , data: InputOnChangeData) => {
-		setReceiver(data.value);
-	};
-
-	const changeValue = (_: any , data: InputOnChangeData) => {
-		checkValue(data.value);
-		setAmount(data.value);
-	};
+	}, [api, isApiReady, receiverAddress]);
 
-	const checkValue = (rawAmount: string) => {
-		const amount = parseAmount(rawAmount);
-		if( !amount || amount.match(/^\d+(,\d+)*(\.\d+)?$/)){
+	useEffect(() => {
+		if( !amount || amount.match(/^\d+(\.\d+)?$/)){
```

Comment:
I realized `ethers` doesn't like commas. still accepting it but parsing and replacing it just like "k" or "m".